docker/docker-entrypoint

```diff
@@ -28,40 +28,42 @@ if [ "$1" == "php-fpm" ] || [ "$1" == "php" ] || [ "$1" == "bin/console" ]; then
         composer dump-env dev
     fi
 
-    echo "Waiting for db to be ready..." 
-    ATTEMPTS_LEFT_TO_REACH_DATABASE=60
-    until [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ] || DATABASE_ERROR=$(bin/console dbal:run-sql "SELECT 1" 2>&1); do
-        if [ $? -eq 255 ]; then
-            # If the Doctrine command exits with 255, an unrecoverable error occurred
-            ATTEMPTS_LEFT_TO_REACH_DATABASE=0
-            break
-        fi
-        sleep 1
-        ATTEMPTS_LEFT_TO_REACH_DATABASE=$((ATTEMPTS_LEFT_TO_REACH_DATABASE - 1))
-        echo "Still waiting for db to be ready... Or maybe the db is not reachable. $ATTEMPTS_LEFT_TO_REACH_DATABASE attempts left"
-    done
+    if [ "$1" == "php-fpm" ]; then
```

Comment:
The if needs to be lower I think. We still want to have a blocking waiting before those services continue to boot further.  That was fine to have.

I think the `if` should only be added around the migration part. So before line 53, and also directly the `fi` after around line 56.